requirements.yml

```diff
@@ -47,7 +47,7 @@
 
 - name: load-balancer
   src: https://github.com/open-craft/ansible-load-balancer
-  version: b729113c24d877aae44246ed4888eb73f773bd19
+  version: 52ae02307d18b7d693113d35523cb4ed045d3ff0
```

Comment:
@pomegranited I'm all in favour of that change.  I didn't like that we are using hashes all along.  We should do that for all roles that are completely under our control.  Otherwise, each PR to a role needs _two_ more PRs to update the requirement hash in the public repo and the submodule hash in the private repo.

Note, however, that you need to use `version: origin/master` for this to work correctly, since ansible-galaxy usually doesn't create or update local remote-tracking branches.